modules/ROOT/pages/authentication-authorization/manage-users.adoc

```diff
@@ -254,7 +256,10 @@ ALTER USER name [IF EXISTS]
   [[SET PASSWORD] CHANGE [NOT] REQUIRED]
   [SET STATUS {ACTIVE \| SUSPENDED} ]
   [SET HOME DATABASE name]
+  [SET AUTH [PROVIDER] 'provider' "{"SET <key> <value>"}"]
   [REMOVE HOME DATABASE]
+  [REMOVE { AUTH [PROVIDER[S]] provider[, ...] } \| { ALL AUTH [PROVIDER[S]] }]
+
 ----
 
 | Description
```

Comment:
in the `Required privilege` part below there is a bit that is `[source, privilege, role="noheader"` and missing it's `]` which makes it look weird when rendered, while you are anyway touching this area it would be nice if you could add the `]` :)

Reply:
strangely, the markup does have the `]` but it is not rendering `[source, privilege, role="noheader"`.   Any idea whats wrong here @renetapopova ?

Reply:
```
[source, privilege, role="noheader"
----
GRANT SET USER STATUS
----
```
no, one of them is missing it

Reply:
ah yes, so it is. thanks. Done.